parsers/US_AK_SAP.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime, timezone
+from logging import Logger, getLogger
+from typing import List, Optional, Union
+
+from requests import Response, Session
+from requests_html import HTMLSession
```

Comment:
Why are we using request_html here when we can get the data by calling the API directly with requests?
https://seapahydro.org/api/scada/index

Reply:
Ah yeah you're definitely right about that one, I should've checked for that. I'm a bit inexperienced at this kind of stuff, sorry.